<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionOpen_UDP"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionSignal_Frame_Play"/>
    <addaction name="separator"/>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionSnapShort"/>
    <addaction name="separator"/>
    <addaction name="actionVideo_Cut_Toolbar"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>PTZ Mode</string>
    </property>
    <addaction name="actionFollowing"/>
    <addaction name="actionLocking"/>
    <addaction name="actionVertical_View"/>
    <addaction name="actionAuto_Fucus"/>
    <addaction name="actionManual_Focus"/>
    <addaction name="actionPTZ_Position_Back"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Scout</string>
    </property>
    <addaction name="actionEnter_Scout_Mode"/>
    <addaction name="actionQuit_Scout_Mode"/>
    <addaction name="actionCircle_Peerpoint"/>
    <addaction name="actionSet_Circle_Distance_to_Peerpoint"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionjian_ti_zhong_wen"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionUDP_Media_Settings"/>
    <addaction name="actionGround_Station_UDP_Settings"/>
    <addaction name="actionAlways_on_Top"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionOption"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="actionOpen_UDP">
   <property name="text">
    <string>Open UDP</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSignal_Frame_Play">
   <property name="text">
    <string>Signal-Frame Play</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionSnapShort">
   <property name="text">
    <string>SnapShort   Ctrl+Shift+LeftButton</string>
   </property>
  </action>
  <action name="actionVideo_Cut_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video Cut Toolbar</string>
   </property>
  </action>
  <action name="actionFollowing">
   <property name="text">
    <string>Following</string>
   </property>
  </action>
  <action name="actionLocking">
   <property name="text">
    <string>Locking</string>
   </property>
  </action>
  <action name="actionVertical_View">
   <property name="text">
    <string>Vertical View</string>
   </property>
  </action>
  <action name="actionAuto_Fucus">
   <property name="text">
    <string>Auto Focus</string>
   </property>
  </action>
  <action name="actionManual_Focus">
   <property name="text">
    <string>Manual Focus</string>
   </property>
  </action>
  <action name="actionPTZ_Position_Back">
   <property name="text">
    <string>PTZ Position Back</string>
   </property>
  </action>
  <action name="actionEnter_Scout_Mode">
   <property name="text">
    <string>Enter Scout Mode</string>
   </property>
  </action>
  <action name="actionQuit_Scout_Mode">
   <property name="text">
    <string>Quit Scout Mode</string>
   </property>
  </action>
  <action name="actionCircle_Peerpoint">
   <property name="text">
    <string>Circle Peerpoint</string>
   </property>
  </action>
  <action name="actionSet_Circle_Distance_to_Peerpoint">
   <property name="text">
    <string>Set Circle Distance to Peerpoint...</string>
   </property>
  </action>
  <action name="actionUDP_Media_Settings">
   <property name="text">
    <string>UDP Media Settings...</string>
   </property>
  </action>
  <action name="actionGround_Station_UDP_Settings">
   <property name="text">
    <string>Ground Station UDP Settings...</string>
   </property>
  </action>
  <action name="actionAlways_on_Top">
   <property name="text">
    <string>Always on Top</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option...</string>
   </property>
  </action>
  <action name="actionjian_ti_zhong_wen">
   <property name="text">
    <string>jian ti zhong wen</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionQuit_Fullscreen_Play">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quit Fullscreen Play</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
